lib/tasks/cortex/core/media.rake

```diff
@@ -29,6 +31,7 @@ namespace :cortex do
                            },
                          metadata:
                            {
+                             naming_data: fieldTitle.id,
```

Comment:
What happened to the original hash we constructed together? It will allow more flexibility with naming later down the road - users of an 'open source' Cortex may want to pass more than one property as an option for filenaming. Cortex is the one place where overengineering is OK, as long as the logic is clear and there's an envisioned need for the company/open source.